### DISCUSSION TITLE
Unexpected loading time for Tailwind css with Next.js in Server-side rendering

### DISCUSSION THREAD

**Discussion by annaczinege** [vercel/next.js#32970] [Help] [answered]
For the website I'm creating I use next.js and for styling tailwind, but can't really see the benefit of server-side rendering that is expected from next.js, because the styles and images have a loading time when you open up the website, (assumingly related to tailwind) and with a slower connection it takes quite a few seconds to get to see the contents of the page.

How could I solve this loading issue and still stick to tailwind?

This is the website link, if you try it, you'll see the few seconds loading time before the content is visible: https://www.revolin.io/

**Comment by wwwdepot** [comment, marked as answer]
Tailwind doesn't involve your site after build time. It compiles the optimized css and it ends there.

SSR is slowing down your site. It makes requests to CMS/API to get the page content on every request.

1. You could opt for [Automatic Static Optimization](https://nextjs.org/docs/advanced-features/automatic-static-optimization). If contents update after build, you could opt for [Incremental Static Generation](https://nextjs.org/docs/basic-features/data-fetching).
2. Your homepage props contains unused data: `termsOfUse` and `privacyPolicy`.

**Reply by annaczinege** [reply]
Thank you!
